examples/rails_app/test/test_helper.rb

```diff
@@ -1,13 +1,15 @@
 ENV['RAILS_ENV'] ||= 'test'
 
+require_relative '../config/environment'
+
 require 'openapi_first'
 OpenapiFirst::Test.setup do |test|
   test.register Rails.root.join('../../spec/data/train-travel-api/openapi.yaml')
   test.register Rails.root.join('../../spec/data/attachments_openapi.yaml'), as: :attachments
+  test.register Rails.root.join('../../spec/data/weather_openapi.yaml'), as: :weather
   test.coverage_formatter_options = { verbose: true }
 end
```

Comment:
@ahx This is probably a breaking change. Because of the `Rails.root undefined` issue the `require_relative '../config/environment` call now needs to be _before_ any `openapi_first` test setup.